aspirateur/zam_aspirateur/amendements/parser.py

```diff
@@ -96,28 +96,51 @@ def parse_bool(text: str) -> bool:
 
 SUBDIV_RE = re.compile(
     r"""^
-        (?:art\.\sadd\.\s(?P<pos>(avant|après))\s)?  # position
-        Article\s
-        (?P<num>\d+|1er)
+        (?:
+            (
+                art\.\sadd\.
+                |
+                Article\(s\)\sadditionnel\(s\)
+            )
+            \s
+            (?P<pos>(avant|après))
+            \s
+        )?  # position
+        article\s
+        (?P<num>\d+|1er|premier)
         (?:\s(?P<mult>\w+))?        # bis, ter, etc.
         (?:\s.*)?                   # junk
         $
     """,
-    re.VERBOSE,
+    (re.VERBOSE | re.IGNORECASE),
 )
 
 
+TITRE_RE = re.compile(r"Titre (?P<num>\w+)(?: .*)?", re.IGNORECASE)
+
+
 def _parse_subdiv(libelle: str) -> Tuple[str, str, str, str]:
     if libelle == "":
         return ("", "", "", "")
-    if libelle.startswith("ANNEXE "):
-        start = len("ANNEXE ")
-        return ("annexe", libelle[start:], "", "")
+
+    if libelle == "Intitulé du projet de loi":
+        return ("titre", "", "", "")
+
+    mo = TITRE_RE.match(libelle)
+    if mo is not None:
+        return "section", mo.group("num"), "", ""
+
+    if libelle.lower().startswith("annexe"):
+        start = len("annexe")
+        return ("annexe", libelle[start:].strip(), "", "")
+
     if libelle.startswith("Chapitre "):
         start = len("Chapitre ")
         return ("chapitre", libelle[start:], "", "")
+
     mo = SUBDIV_RE.match(libelle)
-    if mo is None:
-        raise ValueError(f"Could not parse subdivision {libelle!r}")
-    num = "1" if mo.group("num") == "1er" else mo.group("num")
-    return "article", num, mo.group("mult") or "", mo.group("pos") or ""
+    if mo is not None:
+        num = "1" if mo.group("num").lower() in {"1er", "premier"} else mo.group("num")
+        return "article", num, mo.group("mult") or "", mo.group("pos") or ""
```

Comment:
Same here, I have no strong opinion about that.

Reply:
Maybe use a NamedTuple...